app/src/test/js/mockService.js

```diff
@@ -0,0 +1,26 @@
+var mockService = {
+    getSyncKeysTree : function() {
+        deferred = $q.defer();
```

Comment:
You've stopped using it globally now it looks like, can you define it using `var` to prevent it from leaking into global scope?